dataikuapi/dss/admin.py

```diff
@@ -1136,6 +1138,71 @@ def set_definition(self, definition):
             body = definition)
 
 
+class DSSPersonalApiKey(object):
+    """
+    A personal API key on the DSS instance
+    Do not call that directly, use :meth:`dataikuapi.DSSClient.get_personal_api_key`
+    """
+    def __init__(self, client, id_):
+        self.client = client
+        self.id_ = id_
+
+    ########################################################
+    # Key description
+    ########################################################
+
+    def get_definition(self):
+        """
+        Get the API key's definition
+        
+        :returns: the personal API key definition, as a JSON object
+        """
+        return self.client._perform_json(
+            "GET", "/personal-api-keys/%s" % (self.id_))
+
+    ########################################################
+    # Key deletion
+    ########################################################
+
+    def delete(self):
+        """
+        Delete the API key
+        """
+        return self.client._perform_empty(
+            "DELETE", "/personal-api-keys/%s" % self.id_)
+
+class DSSPersonalApiKeyListItem(DSSTaggableObjectListItem):
+    """An item in a list of datasets. Do not instantiate this class, :meth:`dataikuapi.DSSClient.list_personal_api_keys` or :meth:`dataikuapi.DSSClient.list_all_personal_api_keys`"""
+    def __init__(self, client, data):
+        super(DSSPersonalApiKeyListItem, self).__init__(data)
+        self.client = client
+
+    def to_personal_api_key(self):
+        """Gets the :class:`DSSPersonalApiKey` corresponding to this item"""
+        return DSSPersonalApiKey(self.client, self._data["id"])
+
+    @property
+    def id(self):
+        return self._data["id"]
+    @property
+    def user(self):
+        return self._data["user"]
+    @property
+    def key(self):
+        return self._data["key"]
+    @property
+    def label(self):
+        return self._data["label"]
+    @property
+    def description(self):
+        return self._data["description"]
+    @property
+    def created_on(self):
+        return self._data["createdOn"]
```

Comment:
We tend to return a datetime more and more.
E.g. `dataikuapi.dss.admin.DSSUserActivity.last_successful_login`.